js/imageRunner.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+// this js file holds the image rotation script, every activation is called a run
+
+//Global variables.  I gave up on the global helper object, maybe later
+var numRuns = 0;
+var constructedImages = [];
+var thisRunRandoms = [21, 21, 21];
+var lastRunRandoms = [];
+
+  // constructor function for image objects
+function ImageConstructor(imageName, imagePath, timesShown, clicks) {
+  this.imageName = imageName;
+  this.imagePath = imagePath;
+  this.timesShown = timesShown;
+  this.clicks = clicks;
+  constructedImages.push(this);
+};
```

Comment:
If every image starts out with zero clicks and has been shown zero times, why not just default these attributes to zero instead of passing their values into the constructor? This is not points off or anything, just a note.